src/pages/cli/start/install.mdx

```diff
@@ -23,11 +21,86 @@ import all0 from "/src/fragments/cli-install-block.mdx";
 
 To set up the Amplify CLI on your local machine, you have to configure it to connect to your AWS account.
 
+> If you already have an AWS profile with credentials on your machine, you can skip this step.
+
+> If you are using CLI version <= 10.6.2 go to [Configure CLI <= 10.6.2](#configure-the-amplify-cli-version--1062)
+
+Configure Amplify by running the following command:
+
+```bash
+amplify configure
+```
+
+`amplify configure` will ask you to sign into the AWS Console.
+
+Once you're signed in, Amplify CLI will ask you to create an IAM user.
+
+> Amazon IAM (Identity and Access Management) enables you to manage users and user permissions in AWS. You can learn more about Amazon IAM [here](https://aws.amazon.com/iam/).
+
+```console
+Specify the AWS Region
+? region:  # Your preferred region
+Follow the instructions at
+https://docs.amplify.aws/cli/start/install/#configure-the-amplify-cli
+
+to complete the user creation in the AWS console
+https://console.aws.amazon.com/iamv2/home#/users/create
+```
+
+Navigate to the [IAM User creation page](https://console.aws.amazon.com/iamv2/home#/users/create) if it's not already open.
+
+Enter a User name and click Next. You can name the user anything but we'll call it "amplify-dev".
+
+![image](/images/cli/user-creation/user-name.png)
+
+Select “Attach policies directly” and select “AdministratorAccess-Amplify” as the Permissions policy. Click Next.
+
+![image](/images/cli/user-creation/user-permissions.png)
+
+On the Review page, check that everything looks good and click Create user.
+
+![image](/images/cli/user-creation/user-review.png)
+
+This will redirect to the users list page. Select the user you just created.
+
+![image](/images/cli/user-creation/user-list.png)
+
+On the user details page, navigate to the “Security credentials”, scroll down to “Access keys” and click “Create access keys”
+
+![image](/images/cli/user-creation/create-access-keys.png)
+
+On the next page, select “Command Line Interface”, acknowledge the warning and click Next
+
+![image](/images/cli/user-creation/ack-page.png)
+
+On the next page click create access key. You’ll then see a page with the access keys for the user. Use the copy icon to copy these values to your clipboard, then return to the Amplify CLI.
```

Comment:
```suggestion
On the next page select **Create access key**. You’ll then see a page with the access keys for the user. Use the copy icon to copy these values to your clipboard, then return to the Amplify CLI.
```

click -> select
capitalized "Create" to follow previous references to button text with the same casing